client/lobby/SelectionTab.cpp

```diff
@@ -234,6 +242,7 @@ void SelectionTab::toggleMode()
 		case ESelectionScreen::saveGame:
 			parseSaves(getFiles("Saves/", EResType::SAVEGAME));
 			inputName->enable();
+			inputName->activate();
```

Comment:
Why was this change necessary? Enable should call activate() anyway from what I see

Reply:
This change fixes #2540